lib/helpers.js

```diff
@@ -349,6 +349,11 @@ exports.truncateDecimals = function (number, digits) {
 };
 
 // return true if the the value is not undefined, null, or NaN.
+exports.notNullEmptyOrUndefined = function (val) {
```

Comment:
this name is so long I will likely never use it. what about `exports.valuey` or `exports.hasValue`?